package.json

```diff
@@ -7,39 +7,44 @@
   "author": "jbcpollak <josh@6river.com> (http://6river.com/)",
   "main": "server/server.js",
   "engines": {
-    "node": "5.7.1",
+    "node": "6.2.1",
     "npm": "3.7.2"
   },
   "scripts": {
     "debug": "NODE_ENV=development node debug .",
     "start": "NODE_ENV=development nodemon .",
     "pretest": "eslint .",
-    "test": "NODE_ENV=test istanbul cover node_modules/.bin/_mocha",
-    "test:no-cover": "NODE_ENV=test mocha",
+    "test": "NODE_ENV=test istanbul cover node_modules/.bin/_mocha -- ${MOCHA_OPTS}",
+    "test:no-cover": "NODE_ENV=test mocha -- ${MOCHA_OPTS}",
     "test:debug": "npm run test:no-cover -- --debug-brk",
     "posttest": "nsp check"
   },
   "dependencies": {
-    "compression": "=1.6.2",
-    "cors": "=2.7.1",
-    "helmet": "=2.1.0",
-    "loopback": "=2.28.0",
-    "loopback-boot": "=2.18.1",
-    "loopback-component-explorer": "=2.5.0",
-		"loopback-component-bunyan": "git://github.com/6RiverSystems/loopback-component-bunyan.git#v0.0.3",
-		"loopback-datasource-juggler": "=2.46.0",
-    "serve-favicon": "=2.3.0"
+    "compression": "1.6.2",
+    "cors": "2.7.1",
+    "es6-promisify": "4.1.0",
+    "helmet": "2.1.0",
+    "loopback": "2.29.1",
+    "loopback-boot": "2.20.0",
+    "loopback-component-bunyan": "git://github.com/6RiverSystems/loopback-component-bunyan.git#v0.0.4",
+    "loopback-component-explorer": "2.5.0",
+    "loopback-connector-remote": "1.3.1",
```

Comment:
this also isn't really necessary - its specifying a data store which may or may not be used by a new server. its like having `loopback-connector-postgres` (or mongo) listed